zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -422,6 +422,24 @@ pub async fn test_responses<State, ReadState>(
         .expect("unexpected error in getdifficulty RPC call");
 
     snapshot_rpc_getdifficulty(get_difficulty, &settings);
+
+    let ua1 = String::from("u1l8xunezsvhq8fgzfl7404m450nwnd76zshscn6nfys7vyz2ywyh4cc5daaq0c7q2su5lqfh23sp7fkf3kt27ve5948mzpfdvckzaect2jtte308mkwlycj2u0eac077wu70vqcetkxf");
+    let z_list_unified_receivers =
+        tokio::spawn(get_block_template_rpc.z_list_unified_receivers(ua1))
+            .await
+            .expect("unexpected panic in z_list_unified_receivers RPC task")
+            .expect("unexpected error in z_list_unified_receivers RPC call");
+
+    snapshot_rpc_z_listunifiedreceivers("ua1", z_list_unified_receivers, &settings);
+
+    let ua2 = String::from("u1uf4qsmh037x2jp6k042h9d2w22wfp39y9cqdf8kcg0gqnkma2gf4g80nucnfeyde8ev7a6kf0029gnwqsgadvaye9740gzzpmr67nfkjjvzef7rkwqunqga4u4jges4tgptcju5ysd0");
```

Comment:
Optional: Open a https://github.com/zcash/zcash ticket to add this unified address to their test vectors.